src/components/accordion/AccordionItem.jsx

```diff
@@ -16,12 +16,15 @@ import Link from '../text/Link';
 import Text from '../text/Text';
 import {AccordionContext} from './Accordion';
 
+type PaddingType = 'xxs' | 'xs' | 's' | 'm' | 'l' | 'xl';
```

Comment:
@coderitual do we want to decouple it from Box types or I should import it here?

Reply:
yep, let's keep it decoupled for now 👍